core.py

```diff
@@ -285,7 +288,7 @@ def store_file(fd, filename, md5, arcpath, arcname):
             self.abort(404, 'no such ticket')
         arcpath = os.path.join(self.app.config['upload_path'], ticket_id + '.tar')
 
-        if self.request.get('complete').lower() not in ['1', 'true']:
+        if self.request.GET.get('complete', '').lower() not in ('1', 'true'):
```

Comment:
In this case. ```[]``` and ```()``` are functionally identical. In Python, ```[]``` denotes a mutable array and ```()``` denotes an immutable tuple. The change simply harmonizes the code with the rest of the repo.